backend/package.json

```diff
@@ -35,6 +35,8 @@
         "convict": "6.2.4",
         "dotenv": "16.3.1",
         "fastify": "4.21.0",
+        "fastify-plugin": "4.5.1",
+        "jose": "4.14.4",
```

Comment:
If you don't use this package remove please this dependency. PR should contain only changes connected with current task. This package should be added in task#20